### PR TITLE
Fixes and tweaks to Syndicate medical cyborgs

### PR DESCRIPTION
**Pull request by ghost**
- Some grammar has been changed on the Syndicate cyborg teleporter
- Non-operatives can no longer use the Syndicate cyborg teleporter
- Energy saws actually have an icon now!
- Syndicate medical cyborgs now have an emag
- Syndicate medical cyborgs no longer have a syringe
- Restorative Nanites now heal 5 brute, 5 burn, 5 tox, 15 oxy, 15 brain, and 3 clone damage per tick. These values may need tweaking, but a 50-telecrystal investment ought to be powerful.
- Operative pinpointers are much more optimized and actually point toward the closet operative
- Adds myself to admins.txt

This is merge-ready.

### CODE REVIEW

**Comment by phil235** [comment]
>Syndicate assault cyborgs no longer start with medical equipment

Already fixed by #12028

**Comment by Xhuis** [comment]
I figured that the issue might have been fixed, but my three queries on the IRC either got low-effort responses or none at all. Still, thanks for letting me know.

**Comment by Xhuis** [comment]
Ready for review

**Comment by Xhuis** [comment]
This is a pretty high-priority merge.
@Razharas

File: html/changelogs/Xhuis-SyndiborgFixes.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+ 
+author: phil235
```
Comment (phil235) [inline html/changelogs/Xhuis-SyndiborgFixes.yml]: wrong author...

Comment (Xhuis) [inline html/changelogs/Xhuis-SyndiborgFixes.yml]: That's what I get for copypasting...

**Pull request merged**